binaries/efinix_ti180/Efinix_README.md

```diff
@@ -0,0 +1,32 @@
+# How to use the Titanium Ti180 J484 Binary Files
+
+## v1.3.1
+### For IMX219 image sensor module:
+#### (Raspberry Pi Camera Module v2 or other compatible IMX219 image sensor module)
+1. Connect the IMX219 image sensor module to the P1 connector on the Titanium Ti180 J484 Development Kit.
+2. Insert an SD Card (2GB or above) into the Titanium Ti180 board.
+3. Connect the Titanium Ti180 board with a monitor screen using an HDMI cable.
+4. Power up the Titanium Ti180 board and follow the [steps](https://www.efinixinc.com/docs/efinity-pgm-v3.3.pdf) on page 7.
+5. Upload the desired binary file (e.g. Infinite-ISP_v1.3.1-IMX219.hex) provided in the release on the Titanium Ti180 J484 Development Kit.
```

Comment:
How to upload binary to the Efinix board? This question will arise. Try to answer it in step 5.